tests/no_wrapper/db_conn_tests.py

```diff
@@ -0,0 +1,54 @@
+"""
+Multiprocess tests.
+
+| Copyright 2017-2025, Voxel51, Inc.
+| `voxel51.com <https://voxel51.com/>`_
+|
+"""
+import unittest
+from pymongo.errors import InvalidOperation
+
+import fiftyone.core.odm as foo
+
+
+class GetDbConnTests(unittest.TestCase):
+    def test_get_db_conn(self):
+        # initial connection
+        conn = foo.get_db_conn()
+        doc = conn.datasets.find_one({}, {"_id": 1})
+        self.assertIsNotNone(doc)
```

Comment:
Is there some setup I need to do for this unit test, e.g. run the database? I'm getting a failure on this line that `doc` is `None`.

Reply:
🤦‍♀️  my bad. didn't think anyone running the tests would have an empty datasets collection. rabbit was right. fixed